R/cmdstanr-package.R

```diff
@@ -18,35 +18,14 @@
 #'   algorithms from \R via CmdStan, the shell interface to Stan
 #'   ([mc-stan.org/users/interfaces/cmdstan](https://mc-stan.org/users/interfaces/cmdstan)).
 #'
+#' @includeRmd vignettes/children/comparison-with-rstan.md
+#'
```

Comment:
Allows us to share the same "Comparison with RStan" section in the package doc and vignette without duplicating it.